spec/models/actor_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
+
+RSpec.describe Actor, type: :model do
+  describe "relationships" do
+    it {should have_many :movies}
```

Comment:
Please also include a test for the other association through the joins record like so:
`it { should have_many :movie_actors }`
`it { should have_many(:movies).through(:movie_actors) }`